docs/content/architecture/pgadmin4.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "pgAdmin 4"
+date:
+draft: false
+weight: 900
+---
+
+![pgAdmin 4 Query](/images/pgadmin4-query.png)
+
+[pgAdmin 4](https://www.pgadmin.org/) is a popular graphical user interface that
+makes it easy to work with PostgreSQL databases from both a desktop or web-based
```

Comment:
❓ Since we mention the desktop client, is it worth showing how to set up that (I don't think so) or link to the pgAdmin docs on that (maybe)?

Reply:
I could look something up and add it. We didn't have anything in the old docs, so it might not be worth it.

Reply:
🤔 In the context of this paragraph, I'm inclined to drop the "desktop" mention. The very next sentence says the operator will help with pgAdmin 4, but that's not the case for a desktop install/deployment.

Reply:
Oh yeah, that is weird. The comment is saying pgAdmin works with the desktop app (and I assume it does), but that has nothing to do with us.